serviceWrappers/linux/kLiveController

```diff
@@ -21,48 +21,71 @@
 ###
 # extend callable functions from "functions" script
 ###
+NAME=kLiveController
+LOG_DIR=/var/log
```

Comment:
Jess,
Eran & Guy E. agreed that there will be de literal values for LOG_DIR etc. and that we won't add defaults to live saas repository. That means that there are values for that those parameters defined in the template but if default file exists they are replaced. In production there won't be defaults file so they won't be replaced.